Formula/libfixbuf.rb

```diff
@@ -1,13 +1,15 @@
 class Libfixbuf < Formula
   desc "Implements the IPFIX Protocol as a C library"
   homepage "https://tools.netsa.cert.org/fixbuf/"
-  url "https://tools.netsa.cert.org/releases/libfixbuf-2.4.1.tar.gz"
-  sha256 "8c535d48120b08df1731de709f2dbd2ba8bce568ad64cac34826102caf594d84"
+  url "https://tools.netsa.cert.org/releases/libfixbuf-2.4.2.tar.gz"
+  sha256 "4286d94224a2d9e21937b50a87ee1e204768f30fd193f48f381a63743944bf08"
   license "LGPL-3.0-only"
 
+  # NOTE: This should be updated to check the main `/fixbuf/download.html`
+  # page when it links to a stable version again in the future.
   livecheck do
-    url "https://tools.netsa.cert.org/fixbuf/download.html"
-    regex(%r{releases/libfixbuf[._-]v?(\d+(?:\.\d+)+)\.t}i)
+    url "https://tools.netsa.cert.org/fixbuf2/download.html"
+    regex(/href=.*?libfixbuf[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Comment:
We get an `Unable to get versions` error because the `<a>` tag is split across multiple lines:
```
<div class="p-release-list"><div class="p-release" id="release-2.4.2"><h1><a href="#release-2.4.2">fixbuf Release 2.4.2, 2023-Feb-9</a></h1><div class="p-release-downloads"><h3>Downloads</h3><div class="p-download"><div class="p-download-link"><span class="p-download-title"><a href="#" onclick='
          lgpl3_clickthrough(event,
            "fixbuf",
            "../releases/libfixbuf-2.4.2.tar.gz")
        '>libfixbuf-2.4.2.tar.gz</a></span><span class="p-download-variants"> (982.4KiB)</span></div><div class="p-download-checksums"><p class="p-download-checksum">(SHA256=4286d94224a2d9e21937b50a87ee1e204768f30fd193f48f381a63743944bf08)</p></div></div></div><div class="p-release-changelog"><h3>Changelog</h3>
   <ul>
    <li>
Updated CERT IPFIX elements to include those added in libfixbuf 3.0.  This update does not include the element name changes that were part of that release.
    </li>
   </ul>
  </div></div><div class="p-release" id="release-2.4.1"><h1><a href="#release-2.4.1">fixbuf Release 2.4.1, 2020-Dec-22</a></h1><div class="p-release-downloads"><h3>Downloads</h3><div class="p-download"><div class="p-download-link"><span class="p-download-title"><a href="#" onclick='
          lgpl3_clickthrough(event,
            "fixbuf",
            "../releases/libfixbuf-2.4.1.tar.gz")
        '>libfixbuf-2.4.1.tar.gz</a></span><span class="p-download-variants"> (977.7KiB)</span></div><div class="p-download-checksums"><p class="p-download-checksum">(SHA256=8c535d48120b08df1731de709f2dbd2ba8bce568ad64cac34826102caf594d84)</p></div></div></div><div class="p-release-changelog"><h3>Changelog</h3>
   <ul>
    <li>
Added support for OpenSSL 1.1.0 and removed support for versions prior to 1.0.2.
    </li>
    <li>
Updated CERT IPFIX elements.
    </li>
   </ul>
  </div></div>
```

Maybe this would be better? (I tested this and it works, but I'm unsure if it's the best we can do.)
```suggestion
    regex(/href=.*?libfixbuf[._-]v?(\d+(?:\.\d+)+)\.t/im)
```

Reply:
Oh shoot, I forgot the download links were just `onclick` events. I could have sworn I tested this before creating the PR but evidently not. Thanks for the catch.

I'll update the regex to use a similar approach to `yaf`, which simply uses the opening quotation mark as an anchor.